### ISSUE TITLE
Question: Can a points layer be rendered as lines / arbitrary path

### ISSUE THREAD

**Issue by cedricsam** [mbloch/mapshaper#299]
So, maybe there's a command that does this more simply, but I was wondering whether it was possible to render data as a path, starting from a points layer?

Say, the example given at NACIS last year: http://mapshaper.org/nacis/map.svg

Where the circles would instead be an arrow pointing in a given direction, based on the data. Sounds though like we'd be pushing the limits of svg rendering, which might not be the primary focus of mapshaper?

**Comment by mbloch** [comment]
Are you interested in rendering each point as an arrow or other symbol? with control of variables like color, rotation, and size? I'm interested in exploring this... how would you give mapshaper the basic symbol shape?

**Comment by cedricsam** [comment]
Yup, I would like each point to render as an arrow. I have a feeling that we had talked about that earlier in the year when we looked into the square symbol.

The quick way I think could be to add a parameter (`length` or an `area`) that could be used to size an arbitrary symbol specified the same way the `-shape` command can insert arbitrary shapes, plus maybe some built-in primary shapes (like a square, a line, etc.).

Color would already be handled by fill/stroke. But rotation and size, that actually points to using the `transform` attribute maybe?

What were you thinking about?

**Comment by mbloch** [comment]
I haven't thought about it... entering a string of coordinates seems a bit cumbersome... you might want to embed SVG symbols from .svg files, so you could design the symbol in Illustrator or the equivalent.

**Comment by cedricsam** [comment]
Yah, sounds good. I’ll have a look and try to come up with a PR. Thanks!
